packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -180,8 +192,81 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
         electionData.addressVoted[msg.sender] = true;
     }
 
-    function getNomineeVotes(address nominee) external view override returns (uint) {
```

Comment:
getNomineeVotes is not needed and only applicable on this votes counting methodology. If needed we should expose a way to see all the ballots later